Sources/PointFree/AtomFeed.swift

```diff
@@ -23,6 +28,144 @@ let pointFreeFeed = View<[Episode]> { episodes in
   )
 }
 
+private let episodesFeedView = itunesRssFeedLayout <| View<SubscriberState> { subscriberState in
+  node(
+    rssChannel: freeEpisodeRssChannel,
+    items: items(subscriberState: subscriberState)
+  )
+}
+
+var freeEpisodeRssChannel: RssChannel {
+  let description = """
+Point-Free is a video series about functional programming and the Swift programming language. Each episode \
+covers a topic that may seem complex and academic at first, but turns out to be quite simple. At the end of \
+each episode we’ll ask “what’s the point?!”, so that we can bring the concepts back down to earth and show \
+how these ideas can improve the quality of your code today.
+"""
+  let title = "Point-Free Videos"
+
+  return RssChannel(
+    copyright: "Copyright Point-Free, Inc. \(Calendar.current.component(.year, from: Current.date()))",
+    description: description,
+    image: .init(
+      link: "https://d3rccdn33rt8ze.cloudfront.net/social-assets/pf-avatar-square.jpg",
+      title: title,
+      url: url(to: .home)
+    ),
+    itunes: .init(
+      author: "Brandon Williams & Stephen Celis",
+      categories: [
+        .init(name: "Technology", subcategory: "Software How-To"),
+        .init(name: "Education", subcategory: "Training"),
+        ],
+      explicit: false,
+      keywords: [
+        "programming,development,mobile,ios,functional,swift,apple,developer,software engineering,server,app"
+      ],
+      image: .init(href: "https://d3rccdn33rt8ze.cloudfront.net/social-assets/pf-avatar-square.jpg"),
+      owner: .init(email: "support@pointfree.co", name: "Brandon Williams & Stephen Celis"),
+      subtitle: "Functional programming concepts explained simply.",
+      summary: description,
+      type: .episodic
+    ),
+    language: "en-US",
+    link: url(to: .home),
+    title: title
+  )
+}
+
+private func items(subscriberState: SubscriberState) -> [RssItem] {
+  return Current
+    .episodes()
+    .sorted(by: their({ $0.freeSince ?? $0.publishedAt }, >))
```

Comment:
I wanted to bump all the episodes that have been made free later to the top, but not sure this is working how I expect. need to write some tests.